lib/vdom_blit.ml

```diff
@@ -33,7 +33,8 @@ module Cmd = struct
       | x ->
           let ctx = {send_msg = p; container = elt; after_redraw = after} in
           let rec loop = function
-            | [] -> Mlfi_isdatypes.ffailwith "No command handler found! (%s)" (Obj.Extension_constructor.name (Obj.Extension_constructor.of_val x))
+            | [] -> Printf.ksprintf failwith "No command handler found! (%s)"
+                      (Obj.Extension_constructor.name (Obj.Extension_constructor.of_val x))
```

Comment:
As a general rule, it is better to keep diffs as small as possible to make them easy to review. In particular, whitespace changes should be isolated and treated separately. Of course, in this case, the change is utterly trivial, but _in general_ it is better not to mix whitespace changes with other types of changes.